accord-core/src/main/java/accord/messages/BeginRecovery.java

```diff
@@ -216,19 +230,21 @@ public static class RecoverOk extends RecoverReply
         public final Ballot accepted;
         public final Timestamp executeAt;
         public final PartialDeps deps;
+        public final PartialDeps acceptedDeps; // only those deps that have previously been proposed
```

Comment:
Good place to comment on when/why the distinction matters, but it if it can/should be covered where this is used that is also fine

Reply:
Yep, agreed